app/src/main/java/com/globant/pokemontcgapp/adapter/PokemonTypesAdapter.kt

```diff
@@ -28,11 +30,12 @@ class PokemonTypesAdapter(private val pokemonTypes: List<PokemonType>) : Recycle
 
     override fun getItemCount(): Int = pokemonTypes.size
 
-    class ViewHolder(itemView: View) :
+    class ViewHolder(itemView: View, private val onPokemonTypeClicked: (PokemonType) -> Unit) :
```

Comment:
Marcos, please change the usage of this callback...Try to define a ListenerIterface to represent the click callback.

Reply:
refactored to use listeners interface, thanks